docs/create_your_first_nn.md

```diff
@@ -0,0 +1,118 @@
+which you can think of a "Hello World" dataset in Deep Learning. 
+This dataset contains 70000 images that represent 10 different categories of clothing items. 
+Each example is a 28x28 grayscale image, associated with a single label from 10 classes. Here are the labels and how 
+they map to actual classes: 
+
+| Label        | Class           | 
+| ------------- |:-------------:| 
+| 0      | T-shirt/top | 
+| 1      | Trouser |
+| 2      | Pullover |
+| 3      | Dress |
+| 4      | Coat |
+| 5      | Sandal |
+| 6      | Shirt |
+| 7      | Sneaker |
+| 8      | Bag |
+| 9      | Ankle boot |
+
+And this is what the images themselves look like: 
+![](images/fashion-mnist-sprite.png)
+
+In this tutorial we are going to build and train a neural network that will be able to classify images like these 
+into given categories. 
+
+## Building a Neural Network
+
+Before our neural network can learn from the data, we need to define the structure of the neural network itself. 
+The basic building block of a neural network is a **layer**. To define a neural network, you need to describe 
+what layers it should consist of. The goal of those layers is to capture some data representation aspects during 
+the training phase. With enough data, layers, and training time, neural networks are capable of capturing enough 
+complexity to result in great performance on many tasks. 
+
+Here, our neural network will consist of a few simple layers in a sequence:  
+
+```kotlin
+private val model = Sequential.of(
+    Input(28,28),
+    Flatten(),
+    Dense(300, Activations.Relu),
+    Dense(100, Activations.Relu),
+    Dense(10, Activations.Linear)
+)
+```
+
+First, we define what kind of input will be passed to this neural network. The images in the dataset are greyscale, 
+so we don't need to worry about multiple color channels, and they are 28 x 28 pixels. Thus, an input image can be 
+represented as a two-dimensional array with 28 x 28 elements (with values ranging from 0 to 255).
+
+The first layer is **Flatten()**. This layer simply reformats the data - it transforms the two-dimensional
+ input array into a one-dimensional array with 784 elements (28 * 28 = 784).
+ 
+Next, we've got a sequence of three **Dense()** layers. Dense layers are the most common layers in all kinds
+ of neural network architectures. They are also called fully connected layers because they connect every neuron 
+ in one layer to every neuron in the next layer.
+ 
+For each Dense layer we have specified its size. The first one has 300 neurons (or nodes), the second one has 100 neurons, 
+and the last one has 10 neurons (remember, we have 10 different classes to predict). 
+
+In each Dense layer we have also explicitly specified an activation function to be used. 
+Without activation functions a neural network would simply be passing data through linear nodes and would not be performing 
```

Comment:
public val activation: Activations = Activations.Relu,

Dense and Conv2D layers use Relu by default. 

Defaults are different from Keras